assets/package-lock.json

```diff
@@ -1,10694 +1,6 @@
 {
-  "name": "assets",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
Why we removed this? 🤔

Reply:
@Menubatenny this file should not be changed in this PR. This probably happened as you ran npm install using an older version. Run npm install after upgrading versions and see if there are any changes on this file